inst/@sym/sym2poly.m

```diff
@@ -30,29 +31,22 @@
 %% @group
 %% syms x y
 %% sym2poly(2*x^2 + 3*x - pi, x)
-%%    @result{} (sym) [2  3  -π]  (1×3 matrix)
+%%    @result{} ans =
+%%   2.0000   3.0000  -3.1416
 %% sym2poly(x^2 + y*x, x)
```

Comment:
Do you think we should change the behaviour when x is specified?  I guess that's more consistent behaviour, but its a change from previous Octsympy releases...

Also line 27/28 says we return a row vector of symbolic expressions...